src/cli.js

```diff
@@ -23,6 +23,18 @@ function getProcessEnvs(val) {
   return pEnv;
 }
 
+function checkInternet() {
+    require('dns').lookup('google.com',function(err) {
```

Comment:
@mdh34 with my long-view mirror I can see privacy-sensitive users marching silently but firmly upon us, and shooting us for contacting THE GOOGLES without user consent. What about replacing the lookup to `npmjs.com` instead? To me that's what makes the most sense, since it's needed anyway to install nativefier. Better ideas welcome.